docs/develop/01_sdk/02_cookbook/06_upgrading_to_v0_29/01_backwards_compatibility.md

```diff
@@ -3,32 +3,24 @@ id: v29-backwards-compatibility
 title: Backwards Compatibility / Interoperability with Previous Versions
 ---
 
-Beside the breaking code changes we have identified three data format changes which might affect interoperability.
-The didsign.io and w3n.id already accept bare requests for attestation, which are effectively the new Credentials, so nothing should break there.
+Depending on how exactly your application interacts with other applications, the changed data formats might make them incompatible.
+Supporting only the latest version will result in poor user experience and therefore is not recommended.
+
+We have only identified the Credentials API as definitely having breaking changes. 
```

Comment:
Messaging is affected too, and of course if people archive or exchange data conforming to our interfaces they would also need to translate it at some point. The Credential API is a good example case of how to manage this breaking change in our messaging, but in this document we should also shortly touch upon what people need to consider if they are interfacing with applications directly via messaging. If they push data around any other way I guess they are on their own, but we could at least make them aware.

Reply:
Do we know people or projects doing any of that?

Reply:
I don't have an overview over who does what, I guess Dudley would. But this is also not a letter to our friends...

Reply:
ok yeah that's an idea